### PR TITLE
OXT-1544: Update website with build/QA docs, event and ecosystem resources

### PR DESCRIPTION
**Pull request by stacktrust**
- Create "Events" page and link to 2016 OpenXT Summit
- Rename "History" to "Ecosystems", update list of papers and presentations
- Add build/QA resources to "Development" page, add table of contents
- Add/revise projects on "Related" page, add table of contents
- Add upstream projects to "About" page

Signed-off-by: Rich Persaud <rich.persaud@baesystems.com>

### CODE REVIEW

File: related.md [hunk 1]
```diff
@@ -4,57 +4,268 @@ title: Related
...
+
+### Xen Servers
+
+- [XenServer / Citrix Hypervisor](http://xenserver.org) &middot; [source](https://www.citrix.com/community/citrix-developer/citrix-hypervisor1/), *China, UK*
+- [XCP-ng](https://xcp-ng.org), *France*
+- U.S. NRL Xenon Separation VMM: [2008](http://www-archive.xenproject.org/files/xensummitboston08/XenSummitSpring08.pdf) &middot; [2018](https://www.platformsecuritysummit.com/2018/speaker/kang/PSEC2018-Xenon-Separation-VMM-Myong-Kang.pdf), *USA*
+
+### AIS Redfield
+
+- [Bareflank](https://bareflank.github.io/hypervisor/) hypervisor toolkit, *USA*
```
Comment (jean-edouard) [inline related.md]: Bareflank should probably be in its own category, it's unrelated to Redfield as far I know.

Comment (stacktrust) [reply related.md]: I've renamed this category to "AIS Virtualization" and revised the text for the Redfield entry.

**Comment by stacktrust** [comment]
Changes in the latest update to this PR:

- add Ecosystems links: HPE RoT, DARPA ERI research, manycore OS, MILS
- add Google Scholar links for some researchers with relevant papers
- shorten some text citations to save layout space
- add Related link to Heads
- add Related link to uXen repo in OpenXT, instead of Trammell's unmaintained uXen repo
- improve consistency for link targets in Related summary table: 
(a) date = acquisition news  
(b) startup name  = archive.org mirror
(c) acquiring company name = post-acquisition collateral

**Comment by jean-edouard** [comment]
LGTM, merging soon.

**Pull request merged**